pkg/cmd/adm/restart.go

```diff
@@ -3,155 +3,198 @@ package adm
 import (
 	"context"
 	"fmt"
-	"time"
+	"os"
 
 	"github.com/kubesaw/ksctl/pkg/client"
-	"github.com/kubesaw/ksctl/pkg/cmd/flags"
 	"github.com/kubesaw/ksctl/pkg/configuration"
 	clicontext "github.com/kubesaw/ksctl/pkg/context"
 	"github.com/kubesaw/ksctl/pkg/ioutils"
-
 	"github.com/spf13/cobra"
 	appsv1 "k8s.io/api/apps/v1"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/apimachinery/pkg/util/wait"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	kubectlrollout "k8s.io/kubectl/pkg/cmd/rollout"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
 	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// NewRestartCmd() is a function to restart the whole operator, it relies on the target cluster and fetches the cluster config
+// 1.  If the command is run for host operator, it restart the whole host operator.(it deletes olm based pods(host-operator pods),
+// waits for the new deployment to come up, then uses rollout-restart command for non-olm based - registration-service)
+// 2.  If the command is run for member operator, it restart the whole member operator.(it deletes olm based pods(member-operator pods),
+// waits for the new deployment to come up, then uses rollout-restart command for non-olm based deployments - webhooks)
```

Comment:
I think it would be gold to have similar content in `command.Long` too.

```suggestion
// NewRestartCmd() is a function to restart the whole operator, it relies on the target cluster and fetches the cluster config
// 1.  If the command is run for host operator, it restart the whole host operator.(it deletes olm based pods(host-operator pods),
// waits for the new pods to come up, then uses rollout-restart command for non-olm based - registration-service)
// 2.  If the command is run for member operator, it restart the whole member operator.(it deletes olm based pods(member-operator pods),
// waits for the new pods to come up, then uses rollout-restart command for non-olm based deployments - webhooks)
```

Reply:
updated it